youtube_dl/extractor/generic.py

```diff
@@ -1090,17 +1090,10 @@ def _playlist_from_matches(matches, getter=None, ie=None):
             return _playlist_from_matches(matches, ie='RtlNl')
 
         # Look for embedded (iframe) Vimeo player
-        mobj = re.search(
-            r'<iframe[^>]+?src=(["\'])(?P<url>(?:https?:)?//player\.vimeo\.com/video/.+?)\1', webpage)
-        if mobj:
-            player_url = unescapeHTML(mobj.group('url'))
```

Comment:
I guess `unescapeHTML` here was intentional. Are you sure it's not required in new code?